docs/sources/developers/plugins/build-a-logs-data-source-plugin.md

```diff
@@ -4,19 +4,24 @@ title: Build a logs data source plugin
 
 # Build a logs data source plugin
```

Comment:
should this be a sub page of Create a Grafana plugin?

Reply:
Yes, it will move there in phase 2.